.circleci/config.yml

```
@@ -48,9 +48,9 @@ jobs:
             aws s3 sync _site s3://<< parameters.bucket >> \
             --acl public-read \
             --delete \
-            --exclude "storybook/*" \
             --exclude "*" \
-            --include "*.*"
+            --include "*.*" \
+            --exclude "storybook/*" \
```

Order matters!

---

Fun fact! :sweat_smile: 

Also, prolly shouldn't have that trailing `\`. I'm surprised it works with it there.
```suggestion
            --exclude "storybook/*"
```